commit 99abc321d7a01a70698546f846584118129e3cdc
Author: Kai-Heng Feng <kai.heng.feng@canonical.com>
Date:   Mon Aug 14 20:11:26 2017 -0700

    Input: elan_i2c - add ELAN0608 to the ACPI table
    
    commit 1874064eed0502bd9bef7be8023757b0c4f26883 upstream.
    
    Similar to commit 722c5ac708b4f ("Input: elan_i2c - add ELAN0605 to the
    ACPI table"), ELAN0608 should be handled by elan_i2c.
    
    This touchpad can be found in Lenovo ideapad 320-14IKB.
    
    BugLink: https://bugs.launchpad.net/bugs/1708852
    
    Signed-off-by: Kai-Heng Feng <kai.heng.feng@canonical.com>
    Signed-off-by: Dmitry Torokhov <dmitry.torokhov@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1637_99abc321d7a01a70698546f846584118129e3cdc.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1637_99abc321d7a01a70698546f846584118129e3cdc.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
